OpenRA.Mods.AS/Duplicates/Projectiles/MissileTA.cs

```diff
@@ -418,7 +443,7 @@ static bool IsNearInclineTop(int vFacing, int loopRadius, int predClfDist)
 		// Will missile climb around incline top if bringing vertical facing
 		// down to zero on an arc of radius loopRadius
 		// Calling this function only makes sense when IsNearInclineTop returns true
-		static bool WillClimbAroundInclineTop(int vFacing, int loopRadius, int predClfDist, int diffClfMslHgt)
+		static bool WillClimbAroundInclineTop(int vFacing, int loopRadius, int predClfDist, int diffClfMslHgt, int speed)
```

Comment:
This seems to be reverting a change i did because the checks were complaining about it. Speed here isn't used.